src/sum.js

```diff
@@ -0,0 +1,15 @@
+export default sum
+
+/**
+* Original Source: http://stackoverflow.com/questions/1230233/how-to-find-the-sum-of-an-array-of-numbers
+*
+* This method will sum up all the array elements, assuming the elements are all numbers
+*
+* @param {Array} arr - the arr to be summed up
+*
+*/
+
+function sum(arr) {
+  return arr.reduce(function(a, b) {
```

Comment:
Thanks, @kentcdodds ! I really appreciate the prompt (and very helpful too!) response.
I did the said changes. My commit was successful. 
Did you receive the commit?